usbl_orogen.orogen

```diff
@@ -2,54 +2,63 @@ name "usbl_evologics"
 using_library "usbl_evologics"
 import_types_from "base"
 import_types_from "usbl_evologics/DriverTypes.hpp"
+using_task_library "iodrivers_base"
 
-task_context "Usbl" do
-   property "source_level", "int", 0
-   property "source_level_control", "bool", false
-   property "low_gain", "bool", false
-   property "carrier_waveform_id", "int", 0
-   property "local_address", "int", 1
-   property "remote_address", "int", 2
-   property "cluster_size", "int", 10
-   property "packet_time", "int", 750
-   property "retry_count", "int", 50
-   property "retry_timeout", "int", 1500
-   property "keep_online", "int", 0
-   property "position_enable", "int", 1
-   property "idle_timeout", "int", 120
-   property "sound_speed", "int", 1500
-   property "im_retry", "int", 100
-   property "send_position_to_mobile", "bool", true
-   property "do_surface_user_input", "bool", false
-
-   property "device_string", "/std/string"
-
-   input_port "message_input", "/usbl_evologics/SendInstantMessage"
-   input_port "burstdata_input", "/std/string"
-
-   output_port "message_output", "/usbl_evologics/ReceiveInstantMessage"
-   output_port "burstdata_output", "/std/string"
-   output_port "stats", "/usbl_evologics/DeviceStats"
-   output_port "connection_status", "/usbl_evologics/ConnectionStatus"
-   output_port "position_samples", "/base/samples/RigidBodyState"
-   output_port "do_surface", "bool"
-
+task_context "Task" do
+   subclasses "iodrivers_base::Task"
+   needs_configuration 
+   
+   # TODO Include more parameters 
+   property "change_parameters", "bool", false
```

Comment:
The device story some configuration properties, so it doesn't really need to be configured before use. So if someone really want to change the properties it has to set this property to true. I'm not sure if it's the best way to do that.